Classes/Hooks/RegistrationEmailHookInterface.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace OliverKlee\Seminars\Hooks;
+
+/*
+ * This file is part of the seminars project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+/**
+ * Interface RegistrationEmailHookInterface
```

Comment:
Please also add yourself as an `@author` to the class PHPDoc so you get the credit for this work.

Reply:
And please add a description of the interface and its purpose. We don't need the name of the interface here in the PHPDoc, though.